src/plugins/raven/services/Raven.js

```diff
@@ -201,6 +209,39 @@ export default class Raven {
     });
   }
 
+  /**
+   * @link https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror
+   *
+   */
+  onError(...args) {
+    const state = this.store.getState();
+    const accessToken = getAccessToken(state);
+
+    if (
+      isJwtExpired(accessToken)
+      || window.location.hostname === 'localhost'
+      || this.logStreamRequestCount > 1
```

Comment:
@cohencodes do you still need this logStreamRequestCount check?

Reply:
@efjacobson this is still needed otherwise we get multiple logs for the same error, removing from error boundary didn't resolve that

Reply:
@cohencodes yeah but i think just not logging the first error that happens is tricky - what if the first error is not one that would be double reported?

how about keeping track of the md5 hashes of the stringified payload and, if that hash already exists, dont send again

Reply:
okay i will try this